### PR TITLE
[Fix] broken link and add others guides

### PR DESCRIPTION
**Pull request by Aju100**
Hi there,

I have a fix out the broken links of CONTRIBUTING.md and added the contents for it.  Few changes
- created and updated the guideline of hypertrons-crx-website (CONTRIBUTING.md)
- created GitHub issue and PR templates which are referenced to CONTRIBUTING.md
- created a code of conduct too and updated the [license agreement](https://cla-assistant.io/hypertrons/hypertrons-crx-website) for making contributions.

> Also maintainer should setup issue label bot.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+```shell
+/self-assign
+```
+
+If somebody claims an issue but doesn't follow up for more than two weeks, it's fine to take over it but you should still leave a comment.
+
+### 2. Fork and clone the repository
+
+Visit [hypertrons/hypertrons-crx][repo] repo and make your copy of the repository by **forking** it.
```
Comment (heming6666) [inline CONTRIBUTING.md]: Shoule be `hypertrons/hypertrons-crx-website` here.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+### 8. Push your branch to GitHub
+
+```shell
+git push -f origin branch-name
+```
+
+### 9. Create a Pull Request
+
+In GitHub, send a pull request to `hypertrons:hypertrons-crx`.
```
Comment (heming6666) [inline CONTRIBUTING.md]: The same.

**Review by heming6666 — COMMENTED** [review]
Thank you @Aju100 for your great work!

I will merge it after these typos are fixed.

**Comment by Aju100** [comment]
@heming6666  I have updated and thanks for the feedback.

**Review by heming6666 — APPROVED** [review]
/approve

Thanks @Aju100

**Pull request merged**